front-end/src/components/FriendRequestButton.tsx

```diff
@@ -13,8 +13,8 @@ interface Props {
 export default function FollowRequestButton(props: Props) {
   // TODO: change this to use currentAuthor.url
   const authorUrl = process.env.REACT_APP_API_URL + "/api/author/" + props.currentAuthor?.id + "/followers/" + props.loggedInUser?.authorId + "/";
-  const inboxUrl = process.env.REACT_APP_API_URL + "/api/author/" + props.currentAuthor?.id + "/inbox/";
   const loggedInUserUrl = process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser?.authorId;
+  const followingUrl = loggedInUserUrl + "/following/" + props.currentAuthor?.id + "/";
```

Comment:
This is unrelated, but it might be good to start using format strings in the future:
https://stackoverflow.com/questions/610406/javascript-equivalent-to-printf-string-format

Reply:
Sounds good, I think anything related to URLs and axios requests we should address when we get to making the wrapper for it (Issue #148)